src/compiler/doc/JSDoc.ts

```diff
@@ -81,4 +83,11 @@ export class JSDoc extends Node<ts.JSDoc> {
             removeFollowingNewLines: true
         });
     }
+
+    getStructure() {
+        return callBaseGetStructure<JSDocStructure>({}, this, {
+            description: this.getInnerText() // TODO Not sure about this
```

Comment:
Yeah, JSDocStructure is lacking at the moment. This is perfect for the time being and you can remove the todo.